Makefile

```diff
@@ -32,3 +35,21 @@ docs: tools
 
 tools:
 	GO111MODULE=on go install github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
+
+.SILENT:
+lint:
+	set -e
+	echo lint: Start
+	MODULE_GROUPS=$$(find examples -mindepth 1 -maxdepth 1 -type d)
+	delete=examples/data-sources
+	echo ${array[@]/$delete}
```

Comment:
I get that we need to remove the data sources as they will never pass the lint check. Can you capitalize the variable names?

Reply:
Done, also changed the name on the variables and made the the actual script working :)